afids_utils/io.py

```diff
@@ -0,0 +1,85 @@
+"""Methods for loading and saving nifti files"""
+from __future__ import annotations
+
+import csv
+from os import PathLike
+
+import nibabel as nib
+import numpy as np
+import pandas as pd
+from numpy.typing import NDArray
+
+from afids_utils.exceptions import InvalidFiducialNumberError
+
+AFIDS_FIELDNAMES = [
+    "id",
+    "x",
+    "y",
+    "z",
+    "ow",
+    "ox",
+    "oy",
+    "oz",
+    "vis",
+    "sel",
+    "lock",
+    "label",
+    "desc",
+    "associatedNodeID",
+]
+
+
+def load_nii(
```

Comment:
My preference would be to change this to just min/max normalize an array -- I don't think we should consider that inherently part of loading a NIfTI, and I don't think loading the NIfTI on its own is then worth its own function.

Reply:
I agree with this I think, but I wonder if we even need to have a normalization function or if something has already been implemented in another package.

Reply:
Yeah it's not really inherently anything to do with AFIDs, just something that shows up in a lot of the auto-afids workflows. I'd be mildly positive to the idea of just totally getting rid of this

Reply:
Yeah - it looks like `sklearn` already has a normalization function as part of its `preprocessing` module. Maybe a discussion with @ataha24 and anyone else using this for normalization and if possible to remove this altogether.